models.py

```diff
@@ -70,6 +70,7 @@ def __unicode__(self):
 
 MARKER_TYPE_ACCIDENT = 1
 MARKER_TYPE_DISCUSSION = 2
+UNITED_HATZALA_CODE = 2
```

Comment:
Because one serves the python side while the other is a js constant..
Both of them are improving code readability...